_data/projectfooter.yaml

```diff
@@ -17,25 +21,31 @@ links:
         url: https://github.com/quarkusio
   - title: Get Help
     subfolderitems:
+      - page: Forums
+        url: https://groups.google.com/forum/#!forum/quarkus-dev
       - page: Chatroom
         url: https://quarkusio.zulipchat.com
-      - page: Google&nbsp;Groups
-        url: https://groups.google.com/forum/#!forum/quarkus-dev
-      - page: FAQ
-        url: /faq
 
 more_links:
-  - title: Quarkus is proudly made of a number of projects including ...
+  - title: Quarkus is made of community projects
```

Comment:
This change concerns me a bit as Mark L. specifically asked for it so that we know there are others. I don't mind shortening the text but we need a way to pass this message.

Reply:
Note: it's not blocking, better focus on the CSS issues.